<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inserisci_cliente</class>
 <widget class="QWidget" name="inserisci_cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuovo Dipendente</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(69, 162, 0);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nome</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textNome">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Cognome</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textCognome">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Codice Fiscale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textCodiceFiscale">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Indirizzo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textIndirizzo">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textEmail">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Telefono</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textTelefono">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ok">
          <property name="styleSheet">
           <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
